gravitee-am-reporter/gravitee-am-reporter-kafka/src/main/java/io/gravitee/am/reporter/kafka/audit/KafkaAuditReporter.java

```diff
@@ -34,123 +35,133 @@
 import io.vertx.core.Vertx;
 import io.vertx.kafka.client.producer.KafkaProducer;
 import io.vertx.kafka.client.producer.KafkaProducerRecord;
-import org.apache.kafka.common.serialization.Serializer;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.producer.ProducerConfig;
 import org.apache.kafka.common.serialization.StringSerializer;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.InitializingBean;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.util.StringUtils;
 
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.Properties;
 
 /**
  * @author Florent Amaridon
  * @author Visiativ
  */
+@Slf4j
 public class KafkaAuditReporter extends AbstractService<Reporter> implements AuditReporter, InitializingBean {
 
-  private static Logger LOGGER = LoggerFactory.getLogger(KafkaAuditReporter.class);
+    private static final String SCHEMA_REGISTRY_URL_KEY = "schema.registry.url";
 
-  @Autowired
-  private Vertx vertx;
+    @Autowired
+    private Vertx vertx;
 
-  @Autowired
-  private KafkaReporterConfiguration config;
+    private KafkaReporterConfiguration config;
 
-  @Autowired
-  private GraviteeContext context;
+    @Autowired
+    private GraviteeContext context;
 
-  @Autowired
-  private Node node;
+    @Autowired
+    private Node node;
 
-  private final DtoMapper dtoMapper;
+    private KafkaProducer<String, AuditMessageValueDto> producer;
 
-  private KafkaProducer<String, AuditMessageValueDto> producer;
+    public KafkaAuditReporter(KafkaReporterConfiguration config) {
```

Comment:
it would make sense to include other injected fields (and to make them final), to avoid mixing 2 approaches to injection in 1 class